pkg/hubble/filters/http.go

```diff
@@ -124,6 +124,35 @@ func filterByHTTPPaths(pathRegexpStrs []string) (FilterFunc, error) {
 	}, nil
 }
 
+func filterByHTTPUrl(urlRegexpStrs []string) (FilterFunc, error) {
+	urlRegexps := make([]*regexp.Regexp, 0, len(urlRegexpStrs))
+	for _, urlRegexpStr := range urlRegexpStrs {
+		urlRegexp, err := regexp.Compile(urlRegexpStr)
+		if err != nil {
+			return nil, fmt.Errorf("%s: %v", urlRegexpStr, err)
+		}
+		urlRegexps = append(urlRegexps, urlRegexp)
+	}
+
+	return func(ev *v1.Event) bool {
+		http := ev.GetFlow().GetL7().GetHttp()
+
+		if http == nil || http.Url == "" {
+			return false
+		}
+
+		url = http.Url
+
+		for _, urlRegexp := range urlRegexps {
+			if urlRegexp.MatchString(url) {
+				return true
```

Comment:
`url` here is the `net/url` module imported earlier (L10), this is causing [the following CI error](https://app.travis-ci.com/github/cilium/cilium/builds/262750662#L442-L444):

```
../pkg/hubble/filters/http.go:144:3: use of package url not in selector
../pkg/hubble/filters/http.go:147:29: use of package url not in selector
```

```suggestion
		for _, urlRegexp := range urlRegexps {
			if urlRegexp.MatchString(http.Url) {
				return true
```